src/components/FormGroupV.vue

```diff
@@ -1,8 +1,12 @@
 <template>
   <div class="s-form-group-v">
     <!-- title -->
-    <div class="s-form-group-v__title" :style="titleLayoutStyle">
-      <label>{{ title }}</label>
+    <div
+      class="s-form-group-v__title"
+      :class="{ 's-form-group-v__title--hidden': !title && !tooltip }"
+      :style="titleLayoutStyle"
```

Comment:
@joshua-larks consider putting style first for pretty formatting :smile: 
```
      :style="titleLayoutStyle"
      class="s-form-group-v__title"
      :class="{ 's-form-group-v__title--hidden': !title && !tooltip }"
```

maybe u need `style` last for overrides tho.

Reply:
Can switch